src/test/java/com/google/jenkins/plugins/computeengine/integration/ITUtil.java

```diff
@@ -0,0 +1,276 @@
+    AttachedDisk boot = new AttachedDisk();
+    boot.setBoot(true);
+    boot.setAutoDelete(BOOT_DISK_AUTODELETE);
+    boot.setInitializeParams(
+        new AttachedDiskInitializeParams()
+            .setDiskSizeGb(Long.parseLong(BOOT_DISK_SIZE_GB_STR))
+            .setDiskType(nameFromSelfLink(BOOT_DISK_TYPE))
+            .setSourceImage(BOOT_DISK_IMAGE_NAME));
+    instanceProperties.setDisks(of(boot));
+    instanceProperties.setTags(new Tags().setItems(of(NETWORK_TAGS)));
+    instanceProperties.setMetadata(
+        new Metadata()
+            .setItems(
+                of(
+                    new Metadata.Items()
+                        .setKey(METADATA_LINUX_STARTUP_SCRIPT_KEY)
+                        .setValue(DEB_JAVA_STARTUP_SCRIPT))));
+    instanceProperties.setNetworkInterfaces(
+        of(
+            new NetworkInterface()
+                .setName(NETWORK_NAME)
+                .setAccessConfigs(of(new AccessConfig().setType(NAT_TYPE).setName(NAT_NAME)))));
+    instanceTemplate.setProperties(instanceProperties);
+    return instanceTemplate;
+  }
+
+  static InstanceConfiguration instanceConfiguration(
+      String startupScript,
+      String numExecutors,
+      String jenkinsLabels,
+      Map<String, String> label,
+      boolean createSnapshot,
+      boolean oneShot,
+      String template) {
+    InstanceConfiguration instanceConfiguration =
+        new InstanceConfiguration(
+            NAME_PREFIX,
+            REGION,
+            ZONE,
+            MACHINE_TYPE,
+            numExecutors,
+            startupScript,
+            PREEMPTIBLE,
+            MIN_CPU_PLATFORM,
+            jenkinsLabels,
+            CONFIG_DESC,
+            BOOT_DISK_TYPE,
+            BOOT_DISK_AUTODELETE,
+            BOOT_DISK_IMAGE_NAME,
+            BOOT_DISK_PROJECT_ID,
+            BOOT_DISK_SIZE_GB_STR,
+            false, /* windows */
+            "", /* windowsPasswordCredentialsId */
+            "", /* windowsPrivateKeyCredentialsId */
+            createSnapshot,
+            null, /* remoteFs */
+            new AutofilledNetworkConfiguration(NETWORK_NAME, SUBNETWORK_NAME),
+            EXTERNAL_ADDR,
+            false, /*useInternalAddress */
+            NETWORK_TAGS,
+            SERVICE_ACCOUNT_EMAIL,
+            RETENTION_TIME_MINUTES_STR,
+            LAUNCH_TIMEOUT_SECONDS_STR,
+            NODE_MODE,
+            new AcceleratorConfiguration(ACCELERATOR_NAME, ACCELERATOR_COUNT),
+            RUN_AS_USER,
+            oneShot,
+            template);
+    instanceConfiguration.appendLabels(label);
+    return instanceConfiguration;
+  }
+
+  static Map<String, String> getLabel(Class testClass) {
+    // GCE labels can only be lower case letters, numbers, or dashes
+    // Used to label the nodes created in a given testClass for deletion
+    return ImmutableMap.of(testClass.getSimpleName().toLowerCase(), "delete");
+  }
+
+  private static void deleteIntegrationInstances(
```

Comment:
Suggest modifying the name to clarify that the instances being deleted are from previous runs of the same IT class.